src/molecules/Amount/Amount.native.js

```diff
@@ -0,0 +1,169 @@
+          return 'regular';
+        case 'large':
+          return 'bold';
+        case 'xlarge':
+          return 'bold';
+        case 'xxlarge':
+          return 'bold';
+        case 'xxxlarge':
+          return 'bold';
+        default:
+          return 'bold';
+      }
+    },
+    lineHeight({ size }) {
+      switch (size) {
+        case 'medium':
+          return 'medium';
+        case 'large':
+          return 'medium';
+        case 'xlarge':
+          return 'xlarge';
+        case 'xxlarge':
+          return 'xxlarge';
+        case 'xxxlarge':
+          return 'xxlarge';
+        default:
+          return 'xxlarge';
+      }
+    },
+  },
+  baseText: {
+    alignSelf({ align }) {
+      switch (align) {
+        case 'left':
+          return 'flex-start';
+        case 'center':
+          return 'center';
+        case 'right':
+          return 'flex-end';
+        default:
+          return 'flex-start';
+      }
+    },
+  },
+};
+
+const BaseText = styled(NativeText)`
+  align-self: ${styles.baseText.alignSelf};
+`;
+
+const getFractionalDisplay = (fractionalPart = '') => {
+  switch (fractionalPart.length) {
+    case 0:
+      return '00';
+    case 1:
+      return `${fractionalPart}0`;
+    case 2:
+      return fractionalPart;
+    default:
+      return fractionalPart.slice(0, 2);
+  }
+};
+
+const getLocaleString = (numberString) => {
+  if (Platform.OS === 'android') {
+    // only android needs this polyfill
+    require('intl');
+    require('intl/locale-data/jsonp/en-IN');
+    return new Intl.NumberFormat('en-IN').format(numberString);
+  } else {
+    return parseInt(numberString, 10).toLocaleString('en-IN');
+  }
+};
+
+const Amount = ({ size, align, testID, children }) => {
+  if (isNaN(children)) {
+    throw new Error(`Expected children to be number \n(Eg. "1234", "12.34")`);
+  }
+
+  const currency = '₹';
+
+  const [integerPart, fractionPart] = children.split('.');
+
+  const fractionDisplay = getFractionalDisplay(fractionPart);
+
+  return (
+    <Flex>
+      <BaseText align={align} {...automation(testID)}>
+        <Text
+          color={styles.text.color({ size })}
+          size={styles.text.size({ size })}
+          _weight={styles.text.weight({ size })}
+          _lineHeight={styles.text.lineHeight({ size })} // First text component within nested texts dictate the line height
+        >
+          {`${currency} `}
+        </Text>
+        <Heading color="shade.980" size={size}>
+          {getLocaleString(integerPart)}
+        </Heading>
+        <Text
+          color={styles.text.color({ size })}
+          size={styles.text.size({ size })}
+          _weight={styles.text.weight({ size })}
+        >{`.${fractionDisplay}`}</Text>
+      </BaseText>
+    </Flex>
+  );
+};
+
+Amount.propTypes = {
+  children: PropTypes.string.isRequired,
+  size: PropTypes.oneOf(['medium', 'large', 'xlarge', 'xxlarge', 'xxxlarge']),
+  align: PropTypes.oneOf(['left', 'center', 'right']),
```

Comment:
remove align and make it left aligned as we discussed.

Reply:
I'll raise a separate PR for removing ```align``` from Text & other components. Okay?